xml/System.Net.Http/HttpClient.xml

```diff
@@ -363,9 +363,32 @@ public class GoodController : ApiController
         <ReturnType>System.Net.IWebProxy</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the global Http proxy.</summary>
+        <value>A proxy used by every call that instantiates a <see cref="T:System.Net.HttpWebRequest" />.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks
+This static property determines the default proxy that all <xref:System.Net.Http.HttpClient> instances use if no proxy is set explicitly in the <xref:System.Net.Http.HttpClientHandler> passed through its constructor. 
+
+The default instance returned by this property will initialize following a different set of rules depending on your platform:
+* **For Windows:** Reads proxy configuration from environment variables, if those are not defined, the user's proxy settings.
+
+* **For OSX:** Reads proxy configuration from environment variables, if those are not defined, the system's proxy settings.
+
+* **For Unix:** Reads proxy configuration from environment variables, in case those are not defined, this property initializes a non-configured instance that bypasses all addresses.
```

Comment:
```suggestion
* **For Unix:** Reads proxy configuration from environment variables or, in case those are not defined, this property initializes a non-configured instance that bypasses all addresses.
```

Reply:
Replacing `Unix` for `Linux`, see https://github.com/dotnet/dotnet-api-docs/pull/2822/files#r307043538 

```suggestion
* **For Linux:** Reads proxy configuration from environment variables or, in case those are not defined, this property initializes a non-configured instance that bypasses all addresses.
```